src/bin/runners/github-action-runner.ts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import "reflect-metadata";
+import { Runner } from "@bc/bin/runners/runner";
+import { EntryPoint } from "@bc/domain/entry-point";
+import Container from "typedi";
+import { JobSummaryService } from "@bc/service/job-summary/job-summary-service";
+import { defaultFlowResult } from "@bc/domain/flow";
+
+export class GithubActionRunner extends Runner {
+  constructor() {
+    super(EntryPoint.GITHUB_EVENT);
+  }
+
+  async execute(): Promise<void> {
+    // initialize configuration
+    await this.initConfiguration();
+
+    const jobSummaryService = Container.get(JobSummaryService);
+
+    // execute pre section
+    const preResult = await this.executePre();
+    if (preResult.isFailure) {
+      await jobSummaryService.generateSummary(defaultFlowResult, preResult.output, []);
+      return process.exit(1);
```

Comment:
we should print command producing the error and also node/project name before exiting the execution (this applies to every failure point)